vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/SpreadsheetDropdownFactory.java

```diff
@@ -44,9 +55,37 @@ public Component getCustomEditorForCell(Cell cell,
     return null;
   }
 
+  private ComboBox<AnalysisMethod> createEditorAnalysisComboBox(Spreadsheet spreadsheet,
+      int selectedCellColumnIndex, int selectedCellRowIndex) {
+    ComboBox<AnalysisMethod> editorComboBox = new ComboBox<>();
+    editorComboBox.setClassName("spreadsheet-combo-box");
+    editorComboBox.setItems(AnalysisMethod.values());
+    editorComboBox.setRenderer(createRenderer());
+    editorComboBox.addValueChangeListener(e -> {
+      if (e.isFromClient()) {
+        //We add a whitespace so the value is not auto incremented when the user drags a value
+        Cell createdCell = spreadsheet.createCell(selectedCellRowIndex, selectedCellColumnIndex,
+            e.getValue() + " ");
+        spreadsheet.refreshCells(createdCell);
+      }
+    });
+    return editorComboBox;
+  }
+
+  private Renderer<AnalysisMethod> createRenderer() {
+    StringBuilder stringBuilder = new StringBuilder();
+    stringBuilder.append(
+        "<div class=\"spreadsheet-list-item\"><div>${item.label}</div> <div><vaadin-icon icon='vaadin:question-circle'><vaadin-tooltip slot=\"tooltip\" text=\"${item.description}\"></vaadin-tooltip></div></div>");
+
+    return LitRenderer.<AnalysisMethod>of(stringBuilder.toString())
+        .withProperty("label", AnalysisMethod::label)
+        .withProperty("description", AnalysisMethod::description);
+  }
+
```

Comment:
NIT: I'd strongly recommend implementing a component renderer to avoid having to debug and troubleshoot literal HTML Content which can look similiar how we did it in the batchinformationlayout: